README.md

```diff
@@ -32,6 +32,8 @@ cerberus:
         -    openshift-sdn
         -    openshift-ovn-kubernetes
     cerberus_publish_status: True                        # When enabled, cerberus starts a light weight http server and publishes the status
+    data_inspection: False                               # When enabled, cerberus collects logs, events and metrics of failed components
```

Comment:
We are using kubernetes cli for all our operations except for this as kubectl cluster-info dump doesn't collect logs, events, metrics like "oc adm inspect" command supports, so we need to add a note in the readme as well as the config to enable it only when OpenShift client is supported to run.

Reply:
Added it.